### PR TITLE
Various fix

### PR DESCRIPTION
**Pull request by jourdain**

### CODE REVIEW

File: Sources/Proxy/Representations/SliceRepresentationProxy/index.js [hunk 1]
```diff
@@ -201,7 +247,11 @@ function vtkSliceRepresentationProxy(publicAPI, model) {
 // Object factory
 // ----------------------------------------------------------------------------
 
-const DEFAULT_VALUES = {};
+const DEFAULT_VALUES = {
+  xSlice: 0,
+  ySlice: 0,
+  zSlice: 0,
```
Comment (agirault) [inline Sources/Proxy/Representations/SliceRepresentationProxy/index.js]: Why? A slice representation should have only one slice value.

File: Sources/Common/Core/MatrixBuilder/index.js [hunk 2]
```diff
@@ -128,6 +130,11 @@ class Transform {
     }
     return this;
   }
+
+  identity() {
+    mat4.identity(this.matrix);
```
Comment (agirault) [inline Sources/Common/Core/MatrixBuilder/index.js]: Is this an opengl matrix (column-major) ?
We discussed with @martinken and @thewtex, and we'll probably want to switch matrix library to consolidate our math implementation in vtk-js (itk, vtk, itk.js, vtk.js Math are row-major, but opengl is column-major, which lead to a lot of confusions).

File: Sources/Proxy/Representations/SliceRepresentationProxy/index.js (hunk 1 above)
Comment (jourdain) [reply Sources/Proxy/Representations/SliceRepresentationProxy/index.js]: I kind of agree but needed a backward compatible implementation so latest VTK could be used in Glance. The correct fix for Glance is to do something similar to what you've done with the various configuration but I could not do it with the time I could spend on it yesterday.

So I brought back the previous API at the proxy level so we can upgrade vtk.js for glance for backward compatibility. Once we cleanup Glance with proper configuration management, we could remove those add-on on that slice representation.

Comment (agirault) [reply Sources/Proxy/Representations/SliceRepresentationProxy/index.js]: I managed to do it in my projects by creating multiple name for the distinct representations:
```js
      SliceX: {
        class: vtkSliceRepresentationProxy,
        options: {
          links: [
            { link: 'ColorWindow', property: 'colorWindow' },
            { link: 'ColorLevel', property: 'colorLevel' },
            { link: 'SliceX', property: 'slice' },
          ],
        },
        props: {
          slicingMode: 'X',
        },
      },
      SliceY: {
        class: vtkSliceRepresentationProxy,
        options: {
          links: [
            { link: 'ColorWindow', property: 'colorWindow' },
            { link: 'ColorLevel', property: 'colorLevel' },
            { link: 'SliceY', property: 'slice' },
          ],
        },
        props: {
          slicingMode: 'Y',
        },
      },
      SliceZ: {
        class: vtkSliceRepresentationProxy,
        options: {
          links: [
            { link: 'ColorWindow', property: 'colorWindow' },
            { link: 'ColorLevel', property: 'colorLevel' },
            { link: 'SliceZ', property: 'slice' },
          ],
        },
        props: {
          slicingMode: 'Z',
        },
      },
      Volume: {
        class: vtkVolumeRepresentationProxy,
        options: {
          links: [
            { link: 'ColorWindow', property: 'colorWindow' },
            { link: 'ColorLevel', property: 'colorLevel' },
            { link: 'SliceX', property: 'xSlice' },
            { link: 'SliceY', property: 'ySlice' },
            { link: 'SliceZ', property: 'zSlice' },
          ],
        },
      },
```

File: Sources/Common/Core/MatrixBuilder/index.js (hunk 2 above)
Comment (agirault) [reply Sources/Common/Core/MatrixBuilder/index.js]: Correct, that was more a global comment regarding the matrix in vtkMatrixBuilder. How should I fill it up in setMatrix?

File: Sources/Proxy/Representations/SliceRepresentationProxy/index.js (hunk 1 above)
Comment (jourdain) [reply Sources/Proxy/Representations/SliceRepresentationProxy/index.js]: I know, I told you to do so, but the Glance support several layouts and usage which makes things slightly more complex. Specially with views that can change orientation dynamically.
As I said, not difficult or impossible, but that would require more time (~2 hours).

File: Sources/Common/Core/MatrixBuilder/index.js (hunk 2 above)
Comment (jourdain) [reply Sources/Common/Core/MatrixBuilder/index.js]: Based on your comment, I would have 2 methods which encode the matrix format in their name or via an argument flag.

File: Sources/Proxy/Representations/SliceRepresentationProxy/index.js (hunk 1 above)
Comment (agirault) [reply Sources/Proxy/Representations/SliceRepresentationProxy/index.js]: ok

**Pull request merged**